master/cmd/determined-master/root_test.go

```diff
@@ -92,9 +94,14 @@ webhooks:
 			},
 		},
 	}
+
```

Comment:
I can't do comment above but on line 93 can we change 

```app.kubernetes.io/name``` to something like ```app.kubernetes.io/NaMe``` so this test fails without the code change

Reply:
see new commit for changes